plugins/state_history_plugin/state_history_plugin.cpp

```diff
@@ -512,4 +512,8 @@ void state_history_plugin::handle_sighup() {
    fc::logger::update(logger_name, _log);
 }
 
+const state_history_log_config* state_history_plugin::trace_log_config() const {
```

Comment:
I think this should be removed. Just use `get_trace_log()` and then if not null `->config()`.